west.yml

```diff
@@ -93,12 +94,8 @@ manifest:
     # changes.
     - name: mcuboot
       repo-path: sdk-mcuboot
-      revision: b95e49304a9ee82f83668d51c1d95c6f2faaf983
+      revision: pull/136/head
       path: bootloader/mcuboot
-    - name: mcumgr
```

Comment:
I can't request changes on the commit log, so I'll just put it on this line:

The commit log says:

```
Remove sdk-mcuboot project reference and use
upstream's reference for mcumgr intead.
```

Should say "sdk-mcumgr", not "sdk-mcuboot".